web_src/js/features/repo-projects.js

```diff
@@ -134,7 +134,7 @@ export function initRepoProject() {
         if ($projectColorInput.val()) {
           setLabelColor($projectHeader, $projectColorInput.val());
         }
-        $boardColumn.attr('style', `background: ${$projectColorInput.val()}!important`);
+        $boardColumn[0].setAttribute('style', `background: ${$projectColorInput.val()}!important`);
```

Comment:
```suggestion
        $boardColumn[0].style = `background: ${$projectColorInput.val()} !important`;
```

Maybe it should be like this?